test/integration/testHTAP.js

```diff
@@ -0,0 +1,101 @@
+const assert = require('assert');
+const async = require('async');
+const connOption = require('./connectionOptions').valid;
+const testUtil = require('./testUtil');
+
+// Only the AWS servers support the hybrid table in the GitHub action.
+if (process.env.CLOUD_PROVIDER === 'AWS') {
+  describe('Query Context Cache test', function () {
+    this.timeout(1000000);
+    let connection;
+    
+    before((done) => {
+      connection = testUtil.createConnection(connOption);
+      async.series([
```

Comment:
let's not use async for one action and let's prefere `testUtil.connectAsync` function

Reply:
I replaced that with connectAsync.